site/docs/cloud-providers/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Supported Cloud Providers
+parent: /cloud-providers
+---
+
+## **Self-hosted deploy previews in your cloud or Kubernetes cluster**
+
+
+Preevy allows you to easily provision preview environments for your Docker Compose applications to the cloud provider or Kubernetes cluster you're using:
+   - For AWS: use `aws configure`. See [AWS lightsail credentials configurations](/drivers/aws-lightsail#credentials-configuration).
```

Comment:
IMO these instructions belong in the driver doc.

Reply:
I believe this is the "driver doc". We just renamed it, per our discussion

Reply:
I meant that this paragraph doesn't belong in the index but rather in each driver's specific doc.

Reply:
zevi will fix